ps3t_game_list.php

```diff
@@ -94,14 +94,27 @@
         else 	{
             echo '<B>' . str_replace('%20',' ',strtoupper($gametype[$i])) . '</B><BR>'; }
         
-        $nextpage = 1;
+        //$nextpage = 1;

```

Comment:
Go ahead and remove all the commented out code, we don't need to retain it since we have the commit history.

Reply:
I removed ALL of the commented out code... didn't delete actual comments. :)